blog/_posts/2022-09-06-ads-object-search.md

```diff
@@ -0,0 +1,91 @@
+---
+layout: blog_post
+title: "The ADS Object Search - How to Find Astronomical Objects in the Literature"
```

Comment:
I'd change the hyphen to a colon: `The ADS Object Search: How to Find Astronomical Objects in the Literature`